pkg/cmd/server/start/master_args.go

```diff
@@ -71,16 +71,25 @@ func NewDefaultMasterArgs() *MasterArgs {
 		MasterPublicAddr: flagtypes.Addr{Value: "localhost:8443", DefaultScheme: "https", DefaultPort: 8443, AllowPrefix: true}.Default(),
 		DNSBindAddr:      flagtypes.Addr{Value: "0.0.0.0:53", DefaultScheme: "http", DefaultPort: 53, AllowPrefix: true}.Default(),
 
+		ConfigDir: &util.StringFlag{},
```

Comment:
But you can't control that dir while starting the server.. which is pretty weird